TSC-Charter.md

```diff
@@ -176,10 +187,48 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public with a clear indication given of whether the
+individual voted or abstained.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
+
+Should a vote to remove a member result in a tie (50%/50% split of non
+abstaining participants) the matter shall be referred to a Foundation
+appointed independent third party mediator for arbitration.
+
+At any point during this process, the individual member in question, the
+individuals opening the request, or the TSC Chair may request that the issue
+be referred to a Foundation appointed third party mediator for
+arbitration.
+
+All decisions regarding removal of TSC members is subject to review by the
+Node.js Foundation Board. While the Board will not have the ability to
+overturn or vacate the decision of the TSC, the Board may issue a statement
+requesting that the TSC revisit the decision and recommend that the matter be
+referred to a Foundation appointed third party mediator for arbitration.
+
+All decisions regarding removal of TSC members is subject to review by the body
+of Node.js collaborators. Should no fewer than one-quarter of the current
+Node.js project Collaborators (as defined by the TSC's governance and
+contribution policies) disagree with the TSC vote, the matter shall be referred
```

Comment:
If we say all org collaborators we would need to have a mechanism that allows 1/4 of those to register objections which can be difficult. That's not too say we shouldn't do this, we just need a workable process